src/main/java/by/onliner/util/ListUtil.java

```diff
@@ -0,0 +1,25 @@
+package by.onliner.util;
+
+import org.openqa.selenium.WebElement;
+import java.util.List;
+import java.util.Random;
+
+/***
+ * The class is used to help us to get element from list
+ */
+public class ListUtil {
+
+    /***
+     * Get random web element from list of web elements.
+     * @param list with web elements.
+     * @return web element.
+     */
+    public static WebElement getRandomWebElementFromList(List<WebElement> list, int maxListSize) {
+        Random rand = new Random();
+        for (WebElement selectedElement : list) {
+            selectedElement = list.get(rand.nextInt(maxListSize));
```

Comment:
will list variable be different or the same in each new iteration?

Reply:
*

Reply:
It was different

Reply:
Fixed